django_airavata/apps/admin/static/django_airavata_admin/src/components/admin/group_resource_preferences/ComputePreference.vue

```diff
@@ -202,6 +214,7 @@ export default {
       services.GroupResourceProfileService.retrieve({ lookup: this.id }).then(
         (groupResourceProfile) => {
           this.localGroupResourceProfile = groupResourceProfile;
+          this.userHasWriteAccess = this.localGroupResourceProfile.userHasWriteAccess;
```

Comment:
This line essentially only gets called when reloading the page in the browser, as far as I can tell. You should also handle the case where the groupResourceProfile is passed as a prop and just get the userHasWriteAccess from that.